pxtblocks/fields/field_tilemap.ts

```diff
@@ -276,6 +281,14 @@ namespace pxtblockly {
             }
 
             tilemap.nextId = id + 1;
+
+            const prefix = `${pxt.sprite.TILE_NAMESPACE}.${pxt.sprite.TILE_PREFIX}`;
```

Comment:
what happens when you change the locale of the editor?

Reply:
This is the qualified name that we generate for the TypeScript. It's the same in all languages. If we want the alt text to be translated, we can do that